Fix bugs and add links (#616)
wso2/docs-choreo-dev: en/docs/developer-portal/test-api.md

# Test an API
 
Before using your API in production, you can test it using the integrated console in Choreo. Once you deploy your API in the required environment, an API consumer can use the application credentials, generate an OAuth 2.0 token, and invoke the API to test its functionality by providing values for the required parameters. 

This guide will take you through the steps to test an API in Choreo. 

> 1. Sign in to Choreo Developer Portal at [https://devportal.choreo.dev](https://devportal.choreo.dev). Alternatively, click the Developer Portal link on the Choreo console header. 

Reviewer: ```suggestion
```suggestion
1. Sign in to Choreo Developer Portal at [https://devportal.choreo.dev](https://devportal.choreo.dev). 

    Alternatively, click **Developer Portal** on the Choreo Console header.
```

    ![Developer Portal](../assets/img/developer-portal/manage-applications/developer-portal.png){.cInlineImage-half}

2. Click **APIs**.
3. Select an API you want to test and click **Try Out** on the card. In this case, skip step 4. Alternatively, you can click on the API and follow step 4. 

> 4. Click **Try Out** from the left panel.

Reviewer: ```suggestion
4. In the left pane, click **Try Out**.
```

5. Select the required environment from the **Environment** dropdown.

    ![Test Panel](../assets/img/developer-portal/test-api/test-panel.png){.cInlineImage-half}

> 6. From the dropdown, select the relevant **Subscribed Application** that you subscribed the API to. To learn how to subscribe to an application, see [Manage Subscriptions](../manage-subscription/). 

Reviewer: ```suggestion
6. From the **Subscribed Application** list, select an application you want to trigger when testing the API.

     !!! info
         This list is visible only if at least one application has subscribed to your API. For more information, see [Manage Subscriptions](../manage-subscription/).
```
Suggesting this revision because this way, it is not clear what the name of the dropdown is. Also, refer [1].

Also, a user who reads this guide before reading [Manage Subscriptions](../manage-subscription/) can get confused when he/she does not see this field.

[1]  https://developers.google.com/style/ui-elements#refer-to-ui-elements

7. Click **Get Test Key** on the **Access Token** field to generate a test access token. To learn more about access tokens and generating tokens for production usage, see [Manage Applications](../manage-application/#generate-keys).

8. Next, expand a resource you want to test by clicking on the resource.

    ![Test Resource](../assets/img/developer-portal/test-api/test-resource.png){.cInlineImage-half}

9. Click **Try it Out** to enable the resource to be tested.  
10. Fill in values for the required parameters and click **Execute**.

    ![Test Resource](../assets/img/developer-portal/test-api/test-enabled-resource.png){.cInlineImage-half}

> 11. Once the Choreo executes the request, you can view the response in the console.  

Reviewer: Let's remove "the" in "the Choreo".
